ttnn/cpp/ttnn/operations/ccl/reduce_scatter/reduce_scatter_pybind.cpp

```diff
@@ -55,25 +55,38 @@ void py_bind_reduce_scatter(pybind11::module& module) {
     detail::bind_reduce_scatter(
         module,
         ttnn::reduce_scatter,
-        R"doc(reduce_scatter(input_tensor: std::vector<ttnn.Tensor>, scatter_dim: int, math_op: ReduceType, *, num_links: int = 1, memory_config: Optional[ttnn.MemoryConfig] = None, topology: ttnn.ccl.Topology, num_workers: int = None, num_buffers_per_channel: int = None) -> std::vector<ttnn.Tensor>
+        R"doc(
 
         Performs an reduce_scatter operation on multi-device :attr:`input_tensor` across all devices.
 
         Args:
-            * :attr:`input_tensor` (ttnn.Tensor): multi-device tensor
-            * :attr:`dim` (int)
+            input_tensor (ttnn.Tensor): multi-device tensor
+            dim (int): Dimension to perform operation
 
         Keyword Args:
-            * :attr:`num_links` (int): Number of links to use for the all-gather operation.
-            * :attr:`memory_config` (Optional[ttnn.MemoryConfig]): Memory configuration for the operation.
-            * :attr:`topology` (Optional[ttnn.ccl.topology]): The topology configuration to run the operation in. Valid options are Ring and Linear. Default = Ring
-            * :attr:`num_workers` (int): Number of workers to use for the operation.
-            * :attr:`num_buffers_per_channel` (int): Number of buffers per channel to use for the operation.
+            num_links (int, optional): Number of links to use for the all-gather operation. Defaults to `1`.
+            memory_config (ttnn.MemoryConfig, optional): Memory configuration for the operation. Defaults to `input tensor memory config`.
+            num_workers (int, optional): Number of workers to use for the operation. Defaults to `None`.
+            num_buffers_per_channel (int, optional): Number of buffers per channel to use for the operation. Defaults to `None`.
+            topology (ttnn.Topology, optional): The topology configuration to run the operation in. Valid options are Ring and Linear. Defaults to `ttnn.Topology.Ring`.
+
+        Returns:
+            ttnn.Tensor: the output tensor.
 
         Example:
 
-            >>> tensor = ttnn.from_torch(torch.tensor((1, 2), dtype=torch.bfloat16), device=device)
-            >>> output = ttnn.reduce_scatter(tensor, dim=0)
+            >>> full_tensor = torch.randn([1, 1, 32, 32], dtype=torch.bfloat16)
```

Comment:
The same comment applies as above. but needs to be * 8 * 8 because each input tensor must be fractured 8 ways (num chips) and then we need 8 of those for the mesh tensor (which will fracture by 8 to get the per chip shape under the hood).